### DISCUSSION TITLE
run a method in backend from js after handling form submit

### DISCUSSION THREAD

**Discussion by parmonov98** [livewire/livewire#4917] [3. Feature ideas]
I have a component that contains 2 tabs and they should have summernote editors integrations.
![image](https://user-images.githubusercontent.com/24564577/165222163-4ea1394b-e910-4078-aa51-1d5643c67ed8.png)

I get run onSubmit method in backend on form submit.
but dispatch an event `set-contents`,  so the summernote generated code set to model props in backend
after that I have to call another event from front to backend I can actually save the form data to DB. 
if I run immediatly after dispatching `set-contents` event it won't save the actual data. 
I want  is being able to call a method from JS using @this directive in component itself. 

thanks in advance!
LIVEWIRE is AWESOME!

**Comment by joshhanley** [comment]
@parmonov98 you should be able to use `@this.call('myMethodName')` to call whatever method you need from javascript, or if you are also using Alpine, use `$wire.call('myMethodName')`

Hope this helps!
